system/locales/zh-cn.json

```
@@ -0,0 +1,243 @@
+    "posts_helper_draft": "—— 草稿",
+    "posts_helper_scheduled": "已安排",
+    "posts_helper_pinned": "已置顶",
+
+    "settings_settings": "网站配置",
+    "settings_name": "博客名称",
+    "settings_name_description": "这个名称会出现在标题与SEO搜索引擎中。",
+    "settings_slogan": "标语",
+    "settings_slogan_description": "通常会出现在首页的网站标题下方，类似简短的说明。",
+    "settings_ispublic": "网站开放中",
+    "settings_ispublic_label": "公开网站任何人都可以查看。",
+    "settings_ispublic_descriptrion": "若关闭此选项，任何人都必须登录才能查看网站。",
+    "settings_timezone": "系统时区",
+    "settings_locale": "界面语言",
+    "settings_dateformat": "日期格式",
+    "settings_dateformat_custom": "自定义：",
+    "settings_timeformat": "时间格式",
+    "settings_timeformat_description": "<a class='ts-text is-external-link' href='https://stackoverflow.com/a/20234207/5203951' target='_blank'>點擊此處</a>參閱自訂日期與時間格式的相關資訊。",
+    "settings_about": "关于 Tunalog",
+    "settings_versions": "目前运行的 Tunalog 版本是 v%s（编译版本为 %s）",
+    "settings_madeof": "由来自 <span class='ts-flag is-taiwan-flag is-rounded is-small'></span> 台湾的 <a href='https://twitter.com/YamiOdymel' target='_blank'>Yami Odymel</a> 所开发，当然还有<a href='https://github.com/caris-events/tunalog/graphs/contributors' target='_blank'>贡献者们</a>的爱心❤️。原始码授权方式为 <a href='https://github.com/caris-events/tunalog/blob/master/LICENSE' target='_blank'>MIT</a>，還請盡情地使用、分享或一同改進。Tunalog 由<a href='https://caris.events' target='_blank'>卡莉絲伊繁星</a>共同開發，而卡莉絲伊繁星屬<a href='https://sorae.co' target='_blank'>對空音商事</a>。<p><a class='ts-text is-external-link' href='https://github.com/caris-events/tunalog' target='_blank'>前往 GitHub 查看原始代码</a></p>",
+
+    "tagedit_tagedit": "编辑标签",
+    "tagedit_manage": "标签管理",
+    "tagedit_deletetag": "删除标签",
+    "tagedit_deletetag_description": "此为不可复原的操作！这个标签将会从关系的 %d 篇文章移除。",
+
+    "tags_tags": "创建标签",
+    "tags_name": "名称",
+    "tags_slug": "网址代号",
+    "tags_nameorslug": "名称或网址代号",
+    "tags_description": "简介",
+    "tags_description_description": "有些主题可能会显示此信息。",
+    "tags_list": "标签列表",
+    "tags_notfound": "找不到任何标签",
+    "tags_count": "文章数量",
+
+    "useredit_useredit": "编辑用户",
+    "useredit_avatar": "头像",
+    "useredit_avatar_description": "这个头像可以在 <a class='ts-text is-external-link' href='https://zh-tw.gravatar.com/' target='_blank'>Gravatar</a> 中編輯。",
+
+    "useredit_password_description": "留空表示不做任何修改。",
+    "useredit_bio": "简介",
+    "useredit_bio_description": "支援HTML标签与Javascript代码。",
+    "useredit_manage": "账号管理",
+    "useredit_deleteuser": "删除用户",
+    "useredit_deleteuser_description": "这是不可复原的操作！这个作者的 %d 篇文章将转移给谁？",
+
+    "users_users": "添加用户",
+    "users_list": "用户列表",
+    "users_email": "电子邮箱地址",
+    "users_nickname": "昵称",
+    "users_password": "密码",
+    "users_createdat": "建立日期",
+
+    "wizard_welcome": "欢迎使用 Tunalog👋",
+    "wizard_description": "你只会在初次启动时见到这个页面，这些设定都可以在之后更改。",
+    "wizard_settings": "网站设置",
+    "wizard_settings_description": "最基础的网站文字与显示设置。",
+    "wizard_name_value": "我的 Tunalog 部落格",
+    "wizard_slogan_value": "這是我最ㄅㄧㄤˋ的生活紀錄部落格😲",

```

```suggestion
    "wizard_name_value": "我的 Tunalog 博客",
    "wizard_slogan_value": "这是我最炫的生活记录博客😲",
```

不確定中國使用「部落格」還是「博客」；而 ㄅㄧㄤˋ 的話我想不出來在地詞彙是什麼

---

都有使用，博客用的比較多。

---

請問`ㄅㄧㄤˋ`是 漢語注音法裏的嗎？😊那天刷Threads看到這種注音法了，感覺很有意思就收藏了一份對照表，在這邊都沒聽過這種方法
> https://gist.github.com/windsting/34b843eba206cc54b0d4a1b9fc4a4abd
對照查看了一下，就是拼音bang(四聲) --> 棒 字吧
這邊`最棒`用來形容`最好的，最厲害的`
您覺得這裏改成`bàng`合適還是`棒`呢？

---

ㄅㄧㄤˋ 有點像網際網路剛出來那時，用來說「太酷了」的感覺

意思有點像「最牛的博客」，但這聽起來不是那麼好聽

想不太出來有什麼詞彙